README.md

```diff
@@ -252,6 +252,22 @@ cd /path/to/this/project/kernel-module/
 
 This will generate `picoevb-rdma.ko`.
 
+
+## Building for dGPU
+
+```
+sudo apt update
+sudo apt install build-essential bc
+sudo apt install nvidia-cuda-toolkit (Required to build application.)
+cd /usr/src/nvidia-535.171.04 (Plese update this path as per release)
```

Comment:
how do you get to this path?

Reply:
This path will change as per release so user need to update it.